### PR TITLE
Add feature for individual transform properties

### PR DESCRIPTION
**Pull request by foolip**
The shortest plausible name for this is "translate, scale, and rotate",
but these are concepts that appear also in DOMMatrix and 2D canvas, so
to disambigutate include "CSS" as a prefix. Also include "properties" as
a suffix to make it clear that it's no about the ability to
translate/scale/rotate, which the `transform` property has, but the
indivudal properties.

### CODE REVIEW

File: feature-group-definitions/individual-transforms.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: CSS translate, scale, and rotate properties
```
Comment (captainbrosset) [inline feature-group-definitions/individual-transforms.yml:1]: These seem to be pretty well documented as "individual transform properties". How about using this as the name?

Comment (foolip) [reply feature-group-definitions/individual-transforms.yml:1]: I agree, that's a common naming of these that I've seen. My only concern is that if you know about the `scale` property, you'd have no way of finding it in a list of feature names where it's called "individual transform properties".

Ultimately I think this is an argument for both a structured/normalized name which would probably be "CSS translate, scale, and rotate properties" in this case, and a description which can spell it out.

@ddbeck what name do you think we should pick here?

Comment (ddbeck) [reply feature-group-definitions/individual-transforms.yml:1]: If this is well documented as "individual transform properties" then I think that wins. If we need the disambiguation or search-ability, perhaps we could append it in parentheses: "Individual transform properties (translate, scale, and rotate)"

But maybe we ought to get on with adding a description field, just for our own auto documenting practice.

```yaml
name: Individual transform properties
description: Transform elements with separate `translate`, `rotate`, and `scale` CSS properties.
```

Comment (foolip) [reply feature-group-definitions/individual-transforms.yml]: Looks good to me, I've updated with your suggestion verbatim.

**Review by ddbeck — APPROVED** [review]

**Pull request merged**